sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/UnlockDeleteContent.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Request body of unlock delete API. </summary>
+    public partial class UnlockDeleteContent
```

Comment:
consider renaming to `DataProtectionUnlockDeleteContent` by adding below in `rename-mapping` section of autorest.md
```yaml
UnlockDeleteRequest: DataProtectionUnlockDeleteContent
```

Reply:
Re-generate code to apply the changes in `autorest.md`

Reply:
Actually we added this mapping in autorest.md and for some reason it was able to rename  "UnlockDeleteResponse" to "DataProtectionUnlockDeleteResult" but not "UnlockDeleteContent" to "DataProtectionUnlockDeleteContent".

Any idea why that is happening?

Reply:
@live1206 can you pls help with the correct rename mapping for this

Reply:
Resolved now. Please ignore.